adal/src/main/java/com/microsoft/aad/adal/BasicWebViewClient.java

```diff
@@ -169,7 +169,15 @@ public void onReceivedError(@NonNull final WebView view,
     public void onReceivedError(@NonNull final WebView view,
                                 @NonNull final WebResourceRequest request,
                                 @NonNull WebResourceError error) {
```

Comment:
>error [](http://example.com/codeflow?start=58&length=5)

nit: we can log the error in an else block, if that is of any intereset to us

Reply:
I considered this and decided to log it _always_ since it could be useful for troubleshooting (answer questions likes: 'why did we abort the request?' or 'why _didn't_ we abort the request?').

LMK if you see any issue I may be overlooking

Reply:
i meant to log the actual "error" text..